src/contracts/interfaces/IRegistryCoordinator.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+/**
+ * @title Interface for a contract the coordinates between various registries for an AVS.
+ * @author Layr Labs, Inc.
+ */
+interface IRegistryCoordinator {
+    // DATA STRUCTURES
+    enum Status
+    {
+        // default is inactive
+        INACTIVE,
+        ACTIVE
+    }
+
+    /**
+     * @notice  Data structure for storing info on operators to be used for:
+     * - sending data by the sequencer
+     * - payment and associated challenges
+     */
+    struct Operator {
+        // the id of the operator, which is likely the keccak256 hash of the operator's public key if using BLSRegsitry
+        bytes32 operatorId;
+        // start taskNumber from which the  operator has been registered
+        uint32 fromTaskNumber;
+        // indicates whether the operator is actively registered for serving the middleware or not
+        Status status;
+    }
+
+    /// @notice Returns the bitmap of the quroums the operator is registered for.
+    function operatorIdToQuorumBitmap(bytes32 pubkeyHash) external view returns (uint256);
+
+    /// @notice Returns the stored id for the specified `operator`.
+    function getOperatorId(address operator) external view returns (bytes32);
+
+    /// @notice Returns task number from when `operator` has been registered.
+    function getFromTaskNumberForOperator(address operator) external view returns (uint32);
+
+    /// @notice Returns the registry at the desired index
+    function registries(uint256) external view returns (address);
```

Comment:
probably want a function for the length of this list as well?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/48/commits/7a220382b4fdbe6880cbec7103c3e5db6a036f24